packages/core/src/universal/hooks/useFeatureFlagSubscription/index.ts

```
@@ -0,0 +1,43 @@
+    url,
+    FEATURE_FLAGS_SUBSCRIPTION
+  );
+
+  const processedFlags = useMemo(() => {
+    if (!subscription.data) {
+      return null;
+    }
+
+    return subscription.data.feature_flag.reduce<FeatureFlagsData>(
+      (acc, flag) => ({
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)